doc/ceph-volume/inventory.rst

```diff
@@ -0,0 +1,16 @@
+.. _ceph-volume-lvm-inventory:
+
+``inventory``
+=============
+The ``inventory`` subcommand queries a hosts disc inventory and provides
+hardware information and metadata on every physical device.
```

Comment:
I would've thought that hardware information was the same as the metadata provided from the devices, what is the difference you see to mention both?

Reply:
It also reports logical volumes, which are not really hardware related.